hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/SimpleRpcScheduler.java

```diff
@@ -49,11 +49,37 @@ public class SimpleRpcScheduler extends RpcScheduler implements ConfigurationObs
    */
   private final RpcExecutor metaTransitionExecutor;
 
+  private RpcExecutor bulkloadExecutor;
+
   /** What level a high priority call is at. */
   private final int highPriorityLevel;
 
   private Abortable abortable = null;
 
+  /**
+   * @param conf                    conf
+   * @param handlerCount            the number of handler threads that will be used to process calls
+   * @param priorityHandlerCount    How many threads for priority handling.
+   * @param replicationHandlerCount How many threads for replication handling.
+   * @param metaTransitionHandler   How many threads for meta handling.
+   * @param bulkLoadHandlerCount    How many threads for bulkload handling
+   * @param highPriorityLevel       high Priority Level
+   * @param priority                Function to extract request priority.
+   */
+  public SimpleRpcScheduler(Configuration conf, int handlerCount, int priorityHandlerCount,
```

Comment:
Is the below constructor still valid? In general I suggest we move the code in the below method to this method and make the below method call this method, with bulkLoadHandlerCount = 0, and better also make the below method as deprecated, and remove it in 3.0.0, for example, as it is IA.LimitedPrivate.